src/ReleaseHistory.md

```diff
@@ -392,4 +392,5 @@
 * BUGFIX: Upgrade schemas to latest version (remove `draft-04` from `$schema` property and change `id` to `$id`). This is necessary because the schemas use the `uri-reference` format, which was not defined in draft-04. https://github.com/microsoft/sarif-sdk/issues/1521
 * API BREAKING: The `Init` methods in the Autogenerated SARIF object model classes are now `protected virtual`. This enables derived classes to add additional properties without having to copy the entire code of the `Init` method.
 * BUGFIX: Transformation from SARIF 1.0 to 2.x throws ArgumentOutOfRangeException, if `result.locations` is an empty array. https://github.com/microsoft/sarif-sdk/issues/1526
-* BUGFIX: Add `Result.Level` (and remove `Result.Rank`) for Fortify Converter based on MicroFocus feedback.
\ No newline at end of file
+* BUGFIX: Add `Result.Level` (and remove `Result.Rank`) for Fortify Converter based on MicroFocus feedback.
+* API NON-BREAKING: Provider a SARIF converter for Visual Studio log files.
```

Comment:
>Provider [](start = 20, length = 8)

s/be 'Provide'

Reply:
Fixed.

---
In reply to: [293605399](https://github.com/microsoft/sarif-sdk/pull/1533#discussion_r293605399) [](ancestors = 293605399)